### ISSUE TITLE
Seal of Truth DoT variation

### ISSUE THREAD

**Issue by israelloiolarego** [wowsims/wotlk#2799]
When i simulate with Retri Paladin, the damage of "Holy Vegeance" the DoT damage wich hit every 3 seconds are variating without any stats variation. I tried to see if was because "Glancing hit" but i wont make sense, because the dmg depends only of the Attack Power.

Here the link with my test with 10 Iterations (to be quickly):
https://wowsims.github.io/wotlk/retribution_paladin/#eJzj4vBgXMHIeIGR0YlJgFGIQeo/SxMjW0BOYmVqEQerAIvUXEYufoEzL5gkXsoqsR09xLRnBxMXi8DTh0xcPAJHHjFJPJBVYn4KFrv0ACR26gWTEhtQACTGI7BzP5NEq5wS81kgj1/gNNCYwzxKbOfOgo3hF3j3mEmJE6jyGkQ9v8BhoInvgRYBpUEq2AX69zJJfJIFmv7rCciOXc+ZuBigkEXgGFA7wyKms0xcM5gYRgFWIGRAqo4qVgnGBkZGLzUuBY4vzAJfmJ1EuJg42IGYEYjZgJgJiFmCGDyYpZgkmLsYlQ1MDQyMdA1MgcjI2MjI1NDIwNjY2BDINDQ2NgIyjA0fMUpw3L3GJNBykEni4R4ehReXmDSeX2IyeHqJ6QUj0wdGzlmMDLcYGWYxMSjdYuIEu+NAk4PkrJkgcNLeEuK0C/aKaWBwzd5oM5NCsASrFo1j/kOjAzWMaTh5hCrmkAIgYbb3vYMjhP/DPoExCJxXbLmuLy6wZXEQWef+sErkkz1MhzEYHIbzq8EKrthDYmGlPUTfLXuIeIv9DrnW14E79tmj2vvBPqb/0FeNmP/2YKr/kz1YG9d3+yCwhhf2Z8+AwB80fQMPhAwgLpR1IFYHAN99qLs=

After 30s you can see the DoT dealing: 778.04, 700.24 and 622.43.
But should be constant since there's no buff coming in or out. Note: the talent Vegeance stay up all the time.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Sim Links and Screenshots**
If applicable, add links to your sim settings or screenshots to help explain your problem.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by lime-green** [comment]
@israelloiolarego It's a spell so it can get resisted:

eg:
```
[24.00] [Player (#1)] [Target 1] {SpellID: 31803} [DEBUG] MAP: 4048.9, RAP: 809.0, SP: 1214.7, BaseDamage:585.1, AfterAttackerMods:778.0, AfterTargetMods:778.0, AfterResistances:778.0, AfterOutcome:778.0, AfterPostOutcome:778.0
[24.00] [Player (#1)] Holy Vengeance Tick Target 1 for 778.04. (778.04 Threat)
```

vs

```
[30.00] [Player (#1)] [Target 1] {SpellID: 31803} [DEBUG] MAP: 4048.9, RAP: 809.0, SP: 1214.7, BaseDamage:585.1, AfterAttackerMods:778.0, AfterTargetMods:778.0, AfterResistances:700.2, AfterOutcome:700.2, AfterPostOutcome:700.2
[30.00] [Player (#1)] Holy Vengeance Tick Target 1 for 700.24. (700.24 Threat)
```

Notice the damage is 778 until resistance is applied

**Comment by israelloiolarego** [comment]
Hi! Thank you for reply!

But Holy Damage is Resistible? I thought that wasn't. I mean, there's no Holy Resistance.
________________________________
De: Josh ***@***.***>
Enviado: sábado, 25 de março de 2023 16:05
Para: wowsims/wotlk ***@***.***>
Cc: israelloiolarego ***@***.***>; Mention ***@***.***>
Assunto: Re: [wowsims/wotlk] Seal of Truth DoT variation (Issue #2799)


@israelloiolarego<https://github.com/israelloiolarego> It's a spell so it can get resisted:

eg:

[24.00] [Player (#1)] [Target 1] {SpellID: 31803} [DEBUG] MAP: 4048.9, RAP: 809.0, SP: 1214.7, BaseDamage:585.1, AfterAttackerMods:778.0, AfterTargetMods:778.0, AfterResistances:778.0, AfterOutcome:778.0, AfterPostOutcome:778.0
[24.00] [Player (#1)] Holy Vengeance Tick Target 1 for 778.04. (778.04 Threat)


vs

[30.00] [Player (#1)] [Target 1] {SpellID: 31803} [DEBUG] MAP: 4048.9, RAP: 809.0, SP: 1214.7, BaseDamage:585.1, AfterAttackerMods:778.0, AfterTargetMods:778.0, AfterResistances:700.2, AfterOutcome:700.2, AfterPostOutcome:700.2
[30.00] [Player (#1)] Holy Vengeance Tick Target 1 for 700.24. (700.24 Threat)


Notice the damage is 778 until resistance is applied

**Comment by lime-green** [comment]
@israelloiolarego it appears that way, I checked WCL since I don't have a paladin to test with:

<img width="895" alt="image" src="https://user-images.githubusercontent.com/9436142/227743286-4882df57-4383-4b3f-8729-9a3203744bf7.png">

https://classic.warcraftlogs.com/reports/yqRVzaYgwdNLxCr3#type=damage-done&source=38&ability=53742&fight=18&target=153

**Comment by Watcher7** [comment]
@israelloiolarego 
You're correct that holy resistance is not a stat that can be normally gained by players, but bosses and elite mobs gain a general spell resistance by virtue of being several levels above the player.

Thank you for the report. Closing as not a bug.
